appcontainer/AppContainerModel.ts

```diff
@@ -316,6 +322,10 @@ export class AppContainerModel extends HoistModel {
         return !isEmpty(this.aboutDialogModel.getItems());
     }
 
+    setInitializingLoadMaskMessage(message: string) {
+        this.initializingLoadMaskMessage = message;
+    }
```

Comment:
Sorry as I think I had this in the spec, but realized that we've recently trended more towards exposing `@bindable` properties directly for use by callers - it's reasonable for apps to set them directly, and we can remove the setter here.  That tracks more with some other updates done recently.

To emphasize though that the property is a public API, please add a proper `/** */` doc string, in addition to the type.  Can replace that banner-style block comment

Reply:
Resolved, added docstring explaining the purpose of the bindable.